### ISSUE TITLE
Implement notifications

### ISSUE THREAD

**Issue by MTGPROD** [ppy/osu#8708]
**Describe the new feature:** 

Just implement the notifications when a friend is online or when you receive a message etc. (even when your beatmap being ranked or something like that can be cool) 

**Proposal designs of the feature:**

**Comment by thenerdie** [comment]
Yes, this is one of the things that they are going to add soon (likely)

**Comment by peppy** [comment]
This is quite a large task which is already planned but has no ETA nor ongoing work.

**Comment by MTGPROD** [comment]
You can take advantage of the new integration for notifications to set up this system. (Unless the bancho API doesn't allow it yet)
